cmd/virtbmc/main.go

```diff
@@ -64,6 +65,13 @@ func main() {
 			},
 		},
 		Action: func(cCtx *cli.Context) error {
+			options.BMCUser = os.Getenv("BMC_USERNAME")
+			options.BMCPassword = os.Getenv("BMC_PASSWORD")
+
+			if options.BMCUser == "" || options.BMCPassword == "" {
+				log.Panic("BMC credentials missing: both BMC_USERNAME and BMC_PASSWORD must be provided")
```

Comment:
Please use `logrus.Panic()` or simply `panic()` instead of introducing new dependency.